homeassistant/components/brottsplatskartan/sensor.py

```diff
@@ -45,12 +45,17 @@ async def async_setup_platform(
 
     async_create_issue(
         hass,
-        DOMAIN,
+        HOMEASSISTANT_DOMAIN,
```

Comment:
Won't this mean that we'll overwrite the issue id in the issue registry with different issues?

https://github.com/home-assistant/core/blob/1aecbb9bd5e1cad82e457a690c471cb46bb8aa83/homeassistant/helpers/issue_registry.py#L157

Reply:
Oh, wow, you're absolutely right!